lib/table.py

```diff
@@ -171,20 +168,22 @@ def get_insert_columns(self, join=True):
             self.cleaned_columns = True
         for item in self.columns:
             thistype = item[1][0]
-            if ((thistype != "skip") and (thistype != "combine") and
-                    (self.contains_pk is True or thistype[0:3] != "pk-")):
+            if (thistype != "skip") and (thistype != "combine"):
                 columns += item[0] + ", "
         columns = columns.rstrip(', ')
+        if not create and self.columns[0][0] == 'record_id':
+            columns = columns.lstrip("record_id,").strip()
```

Comment:
This will remove any of the characters in `record_id,` not just `record_id,`.

Would `columns.replace("record_id,", "", 1)` be better? The `1` limits the number of replacements to the first value in case there happened to be another column named `record_id`.